vtex/admin-ui
# Search filter

#### What is the purpose of this pull request?
Add searchable filter

#### How should this be manually tested?
Check storybook for search filters, also this change shouldn't change behaviour for combobox or current filters

#### Screenshots or example usage
<img width="286" alt="image" src="https://user-images.githubusercontent.com/8623116/167708905-e7c7c0af-d7d4-42d3-bd8b-ec23b6421e3d.png">


#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
